### ISSUE TITLE
different cardcolor

### ISSUE THREAD

**Issue by Aeonia** [florent37/MaterialTextField#2]
HI, is it possible to have a separate color for the cardColor( before being pressed) and the Edit text field, after being pressed..setting edittext background doesn't work

**Comment by florent37** [comment]
yes I set null to the edittext background, you want to have, for example, an EditText with black background on a white card ?

or change the color of the card depending on collapsed/expanded ?

**Comment by Aeonia** [comment]
Yes..How can that be achieved?

**Comment by Aeonia** [comment]
Also it doesn't work well with kitkat and below...
Finally, can we change height of the collapsed card?

**Comment by florent37** [comment]
I added an attribute to change the height
and fixed the pre-lollipop on the 1.0.2

**Comment by Aeonia** [comment]
Tried to change EditText's background programmatically...didn't work
[URL=http://s1366.photobucket.com/user/Aeonia92/media/Capture_zps3cf9ddqv.png.html][IMG]http://i1366.photobucket.com/albums/r761/Aeonia92/Capture_zps3cf9ddqv.png[/IMG][/URL]

**Comment by varunagarwal315** [comment]
Im having a similar issue. say I want the label to be blue for which I type app:mtf_labelColor="#0000ff"
but I want the edit text to look like a white box like how it does when i set background of edit text to #FFFFFF but the card outline and bottom line when it is collapsed to be say black. So it looks like an edit text with a white background and a black line surrounding it, with the blue label. Any way to achieve this sir?

**Comment by mridah** [comment]
@florent37 any update on this issue ? even I want a similar thing: white label and red edittext. how can I achieve this ?

Also, is there a way to use this with multiline edittext ? i tried using `minLines` and `lines` in edittext, but it doesn't seem to work
